src/index.js

```diff
@@ -10,7 +10,7 @@ export default function instantMeiliSearch(hostUrl, apiKey, options = {}) {
 
     transformToMeiliSearchParams: function (params) {
       const searchInput = {
-        q: params.query,
+        q: params.query === '' ? null : params.query,
```

Comment:
Does this make it possible to not use placeholder search ?

Reply:
Hum! Algolia enforces the browse, I didn't find how to deactivate it, but we definitely should provide a way to disable it, even if it's not most of the cases.